fairlearn/grid_search/binary_protected_attribute/regression.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import copy
+import numpy as np
+
+from . import utilities
+
+
+def _regression_weight_function(a_val, trade_off, p0, p1, a0_val):
+    if a_val == a0_val:
+        return trade_off / p0
+    else:
+        return (1 - trade_off) / p1
+
+
+def bounded_group_loss(learner,
```

Comment:
As a user without knowledge I don't know what tradeoffs mean in this context. Can we add some documentation for this method?

Also, should it be sweep_ again?

Reply:
It should have `sweep` already. Yes, it needs the same docs as `sweep_demographic_parity` on the classifier side

Reply:
Docs added